lib/signs_ui_web/channels/signs_channel.ex

```diff
@@ -44,6 +44,13 @@ defmodule SignsUiWeb.SignsChannel do
       {:ok, _claims} -> true
       {:error, _error} -> false
     end
+
+    with {:ok, claims} <- SignsUiWeb.AuthManager.decode_and_verify(token, claims),
```

Comment:
Maybe make the `case` expression above part of this `with` chain?

Reply:
Er, actually I think shouldn't it just be deleted?

Reply:
Oops!